packages/runtime/datastore/src/fluidHandle.ts

```diff
@@ -77,7 +77,7 @@ export class FluidObjectHandle<
 	/**
 	 * {@inheritDoc @fluidframework/core-interfaces#IFluidHandle.get}
 	 */
-	public async get(): Promise<any> {
+	public async get(): Promise<T> {
```

Comment:
Unless we're calling it a bug fix (which I could see), this is technically breaking for a legacy+alpha API, right?

Reply:
I was originally leaning towards considering it a bug fix, but I think to be consistent with the changes I chose not to make elsewhere, I'll revert this and leave a TODO.